modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js

```diff
@@ -35,6 +36,7 @@ export default function App({state}) {
 	const initialState = reducer(state, {type: INIT});
 
 	useAutoExtendSession();
+	useIE11Compatibility();
```

Comment:
(Unrelated commentary...) A hook that is defined and used in exactly one place is probably a smell; hooks are about re-use and composability, so if you make a hook to only ever use it once then you are adding a layer of indirection that doesn't really leverage the actual usefulness of hooks.